app/src/main/java/mozilla/lockbox/LockboxApplication.kt

```diff
@@ -21,6 +22,13 @@ val log: Logger = Logger("Lockbox")
 class LockboxApplication : Application() {
     override fun onCreate() {
         super.onCreate()
+        if (LeakCanary.isInAnalyzerProcess(this)) {
+            // This process is dedicated to LeakCanary for heap analysis.
+            // You should not init your app in this process.
+            return
+        }
+        LeakCanary.install(this)
+
```

Comment:
Should this whole block be wrapped in 

```kt
if (BuildConfig.DEBUG) {}
```

Reply:
No, because in `build.gradle` we do the `noop` version of LeakCanary for the `releaseImplementation` configuration.